### ISSUE TITLE
ontop 1.18.0 CLI

### ISSUE THREAD

**Issue by PaulZH** [ontop/ontop#129]
/ontop-1.18.0/ontop-distribution-1.18.0.zip

The ontop shell script needs to point to the renamed it.unibz.inf.ontop.cli.Ontop class.

E.g.

```
${JAVA} ${ONTOP_JAVA_ARGS} -cp "${ONTOP_HOME}/lib/*" -Dlogback.configurationFile="${ONTOP_HOME}/log/logback.xml" \
-Djava.ext.dirs="${ONTOP_HOME}/lib/:${ONTOP_HOME}/jdbc/" it.unibz.inf.ontop.cli.Ontop $@

```

**Comment by ghxiao** [comment]
This has been reported by #126. We are fixing it now.

**Comment by PaulZH** [comment]
Hi Guohui,

Thanks for coming back quickly.

According to me this is not the same.
126 is about the webapp.
129 about Protégé.


Paul

Kind Regards,
Paul Hermans

-------------------------
ProXML bvba
Linked Data services
KBO: http://data.kbodata.be/organisation/0476_068_080#id
(w) www.proxml.be<http://www.proxml.be/>
(e) paul@proxml.be<mailto:paul@proxml.be>
(tw)  @PaulZH
(t)  +32 15 23 00 76
(m) +32 473 66 03 20


Narcisweg 17
3140 Keerbergen
Belgium

ODEdu – Innovative Open Data Education and Training based on PBL and Learning Analytics - http://odedu-project.eu/
OpenGovIntelligence  – Public Administration Modernization by exploiting Linked Open Statistical Data  - http://www.opengovintelligence.eu
OpenCube – Linked Open Statistical Data - http://opencube-project.eu/

**Comment by ghxiao** [comment]
You are right, it is actually #123.

**Comment by ghxiao** [comment]
Fixed in 8d747638346544d4dea9d6f1ed085dd4b33e1cca.

The fixed file is uploaded to source forge as 

/ontop-1.18.0/ontop-distribution-1.18.0.1.zip
